frontend/src/metabase/admin/tasks/components/Logs/Logs.unit.spec.tsx

```diff
@@ -97,6 +97,22 @@ describe("Logs", () => {
         expect(screen.getByText(errMsg)).toBeInTheDocument();
       });
     });
+
+    it("should stop polling on unmount", async () => {
+      fetchMock.get("path:/api/util/logs", [log]);
+      const { unmount } = render(<Logs />);
+      await waitFor(() => {
```

Comment:
```
expect(await screen.findByText(new RegExp(log.process_uuid))).toBeInTheDocument();
```

Reply:
@elton-okawa as soon as you address this last bit, I'll run the full CI with your changes. Thanks for your patience! 🙏

Reply:
@nemanjaglumac I've just addressed it!
It's my pleasure, I'm glad I was able to help and also learn other approaches, thanks!

Reply:
Here we go.
https://github.com/metabase/metabase/pull/45154